ideascube/serveradmin/catalog.py

```diff
@@ -342,7 +342,7 @@ def theme(self):
         # For this we use a dummy function who do nothing.
         # As the function is named _, gettext will mark the strings.
         _ = lambda t: t
-        if '.map' in self.id:
+        if '.map' in self.id or 'maguare' in self.id:
```

Comment:
Yes, we have to find a way to solve this hack.
But for the context of this PR, everything seem ok.